### DISCUSSION TITLE
why?

### DISCUSSION THREAD

**Discussion by leo94666** [neoclide/coc.nvim#3373] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/71742460/132988361-50bd899b-954b-4a38-b14d-7efba1f77843.png)

press gd can go to head file, but this tips?,why?

**Comment by oblitum** [comment, marked as answer]
Your language server points it can't find the header. You have to configure it correctly.

**Reply by leo94666** [reply]
My language server points it can find the header.

**Reply by oblitum** [reply]
The screenshot just says it doesn't. The message comes from the language server.
